golos.publication/parameters.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+#include <eosiolib/singleton.hpp>
+#include <common/parameter.hpp>
+#include <common/parameter_ops.hpp>
+
+namespace golos {
+
+    using namespace eosio;
+
+    struct st_max_vote_changes : parameter {
+        uint8_t max_vote_changes;
+
+        void validate() const override {
+            eosio_assert(max_vote_changes > 0, "Max vote changes can't be negative.");
```

Comment:
remove this assert (and whole `validate()`, because empty one will be inherited). actually, there is no reason to disallow `0` value. and negative values impossible due unsigned type